src/Services/Gateways/PagoEfectivo.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace Ebanx\Benjamin\Services\Gateways;
+
+use Ebanx\Benjamin\Models\Payment;
+use Ebanx\Benjamin\Services\Adapters\CashRequestAdapter;
+
+class PagoEfectivo extends BaseGateway
+{
+    public function create(Payment $payment)
+    {
+        $payment->type = "pagoefectivo";
```

Comment:
Also, the convertion from our gateway based type to pay's direct api "canonical name" should be done in the adapter. Pay's quirks should be invisible to the user.